core/src/main/java/org/apache/struts2/components/Include.java

```diff
@@ -90,27 +92,50 @@
 
     private static final Logger LOG = LogManager.getLogger(Include.class);
 
-    private static String systemEncoding = System.getProperty("file.encoding");
+    private static final String systemEncoding = System.getProperty("file.encoding");
 
     protected String value;
     private HttpServletRequest req;
     private HttpServletResponse res;
-    private static String defaultEncoding;
+    private String defaultEncoding;   // Made non-static (during WW-4971 fix)
```

Comment:
IMO the comments shouldn't contain to many of those ticket references. Therefore we have the git history.

Reply:
I think there are a few spots in the source that refer to ticket references,  After seeing that, it seemed like a good idea to identify the ticket reference for the change via comment.  If the Git history is considered sufficient/preferred those comment references could be eliminated.

Reply:
Yes, this is a good practice and if you feel it is needed that's ok - but we can also depend on git history - your choice :)

Reply:
Yes, this is a good practice and if you feel it is needed that's ok - but we can also depend on git history - your choice :)